about.html

```diff
@@ -69,6 +69,6 @@ <h3 class="clemson-orange-text">Educational Benefits of a CTF</h3>
     </div>
   </div>
   <h3 class="clemson-orange-text">When and Where?</h3>
-  <p>CUCTF will take place October 3rd, 2020, hosted at Clemson's most technologically advanced building: The Watt Center.</p>
+  <p>CUCTF 2020 will take place <strong>October 3rd, 2020</strong>. While normally hosted at Clemson University's Watt Family Innovation Center, <strong>this event will be hosted online</strong>. More information on competing remotely will be announced soon! Please fill out our <a href="register.html">interest form</a> to be updated as updates come out! </p>
```

Comment:
Change link location to be absolute (change to /register) and then it's good imo